service/controller/flannel.go

```diff
@@ -27,7 +29,11 @@ type FrameworkConfig struct {
 	ProjectName  string
 }
 
-func NewFramework(config FrameworkConfig) (*controller.Controller, error) {
+type Flannel struct {
+	*controller.Controller
+}
```

Comment:
I just named it flannel.

Reply:
The concept being managed might be `network`.

Reply:
Renamed to `network`